src/fs.zig

```diff
@@ -108,6 +108,11 @@ pub const FileSystem = struct {
         return &instance;
     }
 
+    pub fn deinit(this: *const FileSystem) void {
+        this.dirname_store.deinit();
```

Comment:
lol i would be surprised if this doesn't still leak

Reply:
if any other fields leak, theyll be found as more tests are checked